docs/coordinates/stokescoord.rst

```diff
@@ -0,0 +1,86 @@
+.. _stokes-coord:
+
+Using the StokesCoord Class
+***************************
+
+The `.StokesCoord` class provides a minimal wrapper for interacting with Stokes parameters as handled by WCS.
+In WCS (as described in the FITS 4.0 specification), the stokes parameters are allocated an integer number, each representing a different parameter, for example "I" is given the number 1 and "Q" the number 2.
```

Comment:
Nit: try to keep line length < 80 or 90 or so.

Reply:
I do that on .tex documents that I use git with indeed...

I think the only argument against it here is that we cannot do that for docstrings, and like those, this text should be nice & readable also if opened as plain text...  Anyway, good to stick with what's here...